<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../images/IMG_3253.ico</normaloff>../images/IMG_3253.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="Table_widgets">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>500</width>
      <height>300</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>경기천년바탕 Regular</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#ffffff

</string>
    </property>
    <column>
     <property name="text">
      <string>시간</string>
     </property>
     <property name="font">
      <font>
       <family>경기천년바탕 Regular</family>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>카테고리</string>
     </property>
     <property name="font">
      <font>
       <family>경기천년바탕 Regular</family>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>내용</string>
     </property>
     <property name="font">
      <font>
       <family>경기천년바탕 Regular</family>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>달성</string>
     </property>
     <property name="font">
      <font>
       <family>경기천년바탕 Regular</family>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QGroupBox" name="Cat_profiles_box">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>540</y>
      <width>450</width>
      <height>230</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>경기천년바탕 Regular</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#ffffff
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QPushButton" name="Cat_profile_btn_1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>75</width>
       <height>75</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Cat_profile_btn_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Cat_profile_btn_3">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Cat_profile_btn_4">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>30</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Cat_profile_btn_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Cat_profile_btn_6">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>130</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Cat_profile_btn_7">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>130</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Cat_profile_btn_8">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>130</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="set_sche_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>820</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Lookup_schedule_btn">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>820</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="quest_label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>490</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>경기천년바탕 Regular</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>오늘의 퀘스트</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="date_tracker">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;경기천년바탕 Regular&quot;;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <action name="action_sched_check">
   <property name="text">
    <string>일정 확인하기</string>
   </property>
  </action>
  <action name="action_sched_add">
   <property name="text">
    <string>일정 추가하기</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
